lib/ui/homepage.dart

```diff
@@ -121,25 +122,26 @@ class _HomePageState extends State<HomePage> {
               flex: 3,
             ),
             Center(
-              child: Row(
+              child: Wrap(

```

Comment:
This was the layout switch I made for the main buttons to do the "right" thing on landscape and portrait.

Reply:
I have basically the same change locally. That happens if you dont push ^^